apps/src/code-studio/initApp/project.js

```diff
@@ -523,7 +523,7 @@ var projects = module.exports = {
         this.sourceHandler.setInitialLevelSource(currentSources.source);
         this.showMinimalProjectHeader();
       }
-    } else if (appOptions.isLegacyShare && this.getStandaloneApp()) {
+    } else if (appOptions.legacyShareStyle && this.getStandaloneApp()) {
```

Comment:
This is the only one I'm not certain on. Are we trying to determine if we're a `/c` share URL here? We're outside of the `this.isProjectLevel()` check.

Reply:
The comment on `showMinimalProjectHeader()` reads
```js
// Minimal project header for viewing channel shares and legacy /c/ share pages.
```
which I _think_ means you've got it right - we want the minimal header in both cases.